### DISCUSSION TITLE
Can I install EMQX on Almalinux 8?

### DISCUSSION THREAD

**Discussion by ramzi03** [emqx/emqx#12258] [Q&A]
Hi guys, I have a droplet with Almalinux 8 on Digital Ocean, can I install EMQX on that droplet? or should I create another droplet with OS that supported as in the docs?
Because when I see the docs, there's only Debian, Ubuntu or CentOS/RHEL. 
Thank you.

**Comment by id** [comment]
Hi @ramzi03. If I understand correctly, AlmaLinux is binary compatible with RHEL, so you could try to use packages for RHEL 8, although we have not tested them with AlmaLinux specifically.
